### ISSUE TITLE
3. Backend Design

### ISSUE THREAD

**Issue by StanislawHornaGitHub** [IIS-ZPI/ZPI2024_zaoczni_NBP_Crasher#29]

**Comment by rukaSUKII** [comment]
@abienias97 Please share your opinion about  options and possibilities:

After checking the NBP API documentation our suggested solution for the system seems to be possible to develop.
It provides all data required to perform analysis specified in requirements specification.
REST API for the purpose of presentation layer should not be complex as there are not much scenarios to cover by it.

**Comment by abienias97** [comment]
I agree with all of the above, FastAPI should be more than capable to handle all the logic of the backend layer.

**Comment by GringoXY** [comment]
@rukaSUKII @abienias97 
For me it seems that we can close the issue. Please let me know whether I can mark it as done ✔

**Comment by rukaSUKII** [comment]
@GringoXY I confirm - this issue can be closed from my point of view
